Marlin/temperature.cpp

```diff
@@ -1539,42 +1539,41 @@ ISR(TIMER0_COMPB_vect) {
       #else
         #define GE0 >=
       #endif
-      if (current_temperature_raw[0] GE0 maxttemp_raw[0]) max_temp_error(0);
-      if (minttemp_raw[0] GE0 current_temperature_raw[0]) min_temp_error(0);
+      if ((current_temperature_raw[0]) GE0 (maxttemp_raw[0])) max_temp_error(0);
+      if ((minttemp_raw[0]) GE0 (current_temperature_raw[0])) min_temp_error(0);
```

Comment:
Mainly because it begun with:
    ... 0.tmp\temperature.cpp:1553:27: error: expected ')' before 'GE0'
       if (minttemp_raw[1] GE0 current_temperature_raw[1]) min_temp_error(1);

When EXTRUDERS = 2
But that did not help. Than discovered the GEO-error. That fixed i got a reading of 478° for temp1 but no MAXTEMPERROR. That was caused by a undefined TEMP_SENSOR_1. ...
and forgot about the ().